minishift-ci-index.yaml

```diff
@@ -145,3 +146,9 @@
           ci_project: '{name}'
           ci_cmd: '/bin/bash centos_ci.sh'
           timeout: '30m'
+      # Docs builds
+      - '{git_repo}-docs':
+          git_repo: minishift
+          ci_project: '{name}'
+          ci_cmd: '/bin/bash centos_ci_docs.sh'
```

Comment:
Do you really want to create a new script here? Most of the parts will be identical, right?